templates/app/cpp/tizen-manifest-service.xml.tmpl

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="{{androidIdentifier}}" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="common"/>
+    <service-application appid="{{androidIdentifier}}" exec="runner" type="capp" multiple="false" nodisplay="true" taskmanage="false" auto-restart="false">
```

Comment:
Probably this makes no issue, but in [docs](https://developer.tizen.org/ko/development/tizen-studio/native-tools/configuring-your-app/manifest-text-editor?langredirect=1#service) there is no 'nodisplay' member in case of service-application tag, but probably it will be just ignored.

Reply:
It is added automatically when creating service application by Tizen Studio.